src/basic-syntax/static-and-const.md

```diff
@@ -1,10 +1,11 @@
 # Static and Constant Variables
 
-Global state is managed with static and constant variables.
+Static and constants are two kind of variables with special constraints.
```

Comment:
This changes the sentence from something concrete:

* Global state is managed by
  * static and
  * constant variable

to something more loose:

* _two kind_ — are there more kinds?
* _special constraints_ — which constraints? How are the constraints special compared to other variables?